.ci/Jenkinsfile

```diff
@@ -93,6 +100,92 @@ pipeline {
         }
       }
     }
+    stage('Publish Snapshot') {
+      environment {
+        SNAPSHOT = 'true'
+      }
+      steps {
+        withGithubNotify(context: "Publish-Snapshot") {
+          dir("${BASE_DIR}"){
+            withMageEnv(version: "${env.GO_VERSION}"){
+              cmd(label: 'Go build', script: 'mage -v build:all')
+            }
+          }
+          publishArtifacts(type: 'snapshot')
```

Comment:
Artifacts are now available in the google bucket

<img width="1068" alt="image" src="https://user-images.githubusercontent.com/2871786/186396516-2c1d2594-2688-4fea-b090-6d38bb98092e.png">

Reply:
where can I find these? I'd like to see result of rekicked run

Reply:
Those artifacts are available in the Google bucket, as we have moveod this project to `fleet-ci` is not anymore available in the `beats-ci` but `fleet-ci` google bucket

```bash
gsutil list gs://fleet-ci-artifacts/jobs/elastic-agent-shipper/commits
```